haystack/document_stores/sql.py

```diff
@@ -20,24 +20,22 @@
 class ORMBase(Base):
     __abstract__ = True
 
-    id = Column(String(100), default=lambda: str(uuid4()), primary_key=True)
+    id = Column(String(100), default=lambda: str(uuid4()), unique=True, primary_key=True)
```

Comment:
Interesting never ever thought needed to set a unique flag along with the primary key, Strange

Reply:
Yeah I had the same reaction :smile: But this _somehow_ fixes the issue reported in the issue. I've yet to really figure out how.